Rakefile

```diff
@@ -1,6 +1,7 @@
 require "yast/rake"
 
-Yast::Tasks.submit_to :sle12sp1
+yast_submit = ENV["YAST_SUBMIT"] || :sle12sp1
+Yast::Tasks.submit_to(yast_submit.to_sym)
```

Comment:
But YAST_SUBMIT already **is** used: https://github.com/yast/yast-rake/blob/2076875ee1c72dd9b4e16b9bb9e00a1c60c4e52b/lib/yast/rake.rb#L29
Except it has Factory as the default. But if you override the variable anyway, maybe you can simply remove this here?